rotkehlchen/tests/api/test_bitcoin.py

```diff
@@ -56,28 +59,26 @@
     '1DNg5csVnUkheYczaBECsfby8ZcmZkYnJK',
 ]
 
+TEST_BITCOIN_XPUB_1 = 'xpub6DCi5iJ57ZPd5qPzvTm5hUt6X23TJdh9H4NjNsNbt7t7UuTMJfawQWsdWRFhfLwkiMkB1rQ4ZJWLB9YBnzR7kbs9N8b2PsKZgKUHQm1X4or'  # noqa: E501
+TEST_BITCOIN_XPUB_2 = 'xpub68V4ZQQ62mea7ZUKn2urQu47Bdn2Wr7SxrBxBDDwE3kjytj361YBGSKDT4WoBrE5htrSB8eAMe59NPnKrcAbiv2veN5GQUmfdjRddD1Hxrk'  # noqa: E501
+
+
+def _check_xpub_addition(rotki: Rotkehlchen, xpubs: List[str]):
+    """Checks the rotki state after xpub addition"""
+    assert rotki.chain_manager.accounts.btc[:2] == [UNIT_BTC_ADDRESS1, UNIT_BTC_ADDRESS2]
+    assert set(rotki.chain_manager.accounts.btc[2:]) == set(EXPECTED_XPUB_ADDESSES)
+
+    with rotki.data.db.conn.read_ctx() as cursor:
+        xpub_data = rotki.data.db.get_bitcoin_xpub_data(cursor)
+    assert [data.xpub.xpub for data in xpub_data] == xpubs
+    assert xpub_data[0].derivation_path is None
 
-def _check_xpub_addition_outcome(outcome, xpub):
-    """Checks the outcome of the xpub additions for the following test.
-    Both results should be the same since the 2nd xpub derives no mainnet addresses
-    """
-    btc = outcome['per_account']['BTC']
-    assert len(btc['standalone']) == 2
-    assert UNIT_BTC_ADDRESS1 in btc['standalone']
-    assert UNIT_BTC_ADDRESS2 in btc['standalone']
-
-    assert len(btc['xpubs']) == 1
-    xpub_data = btc['xpubs'][0]
-    assert xpub_data['xpub'] == xpub
-    assert xpub_data['derivation_path'] is None
     for address in EXPECTED_XPUB_ADDESSES:
-        assert address in xpub_data['addresses']
-        assert xpub_data['addresses'][address]['amount'] is not None
-        assert xpub_data['addresses'][address]['usd_value'] is not None
+        if address == '1KZB7aFfuZE2skJQPHH56VhSxUpUBjouwQ':
```

Comment:
Why add this exception? Comment why so reader understands ... but first tell me why. Where did the test change?